org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/server/wizard/CustomWizardDialog.java

```diff
@@ -19,23 +19,34 @@
  */
 package org.sonarlint.eclipse.ui.internal.server.wizard;
 
-import javax.annotation.Nullable;
-import org.sonarlint.eclipse.core.internal.TriggerType;
-import org.sonarlint.eclipse.core.internal.server.IServer;
-import org.sonarlint.eclipse.ui.internal.server.actions.JobUtils;
+import org.eclipse.jface.wizard.WizardDialog;
+import org.eclipse.swt.widgets.Shell;
 
-public class EditServerLocationWizard extends AbstractServerLocationWizard {
+public class CustomWizardDialog extends WizardDialog {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This class has 6 parents which is greater than 5 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AMaximumInheritanceDepth)